docs/src/terminology.md

```diff
@@ -130,13 +134,15 @@ A [public key](terminology.md#public-key) and corresponding [private key](termin
 
 A fractional [native token](terminology.md#native-token) with the value of 0.000000001 [sol](terminology.md#sol).
 
+The smallest unit of the [native token](terminology.md#native-token), SOL. Equal to 0.000000001 [SOL](terminology.md#sol). Named after computer scientist [Leslie Lamport.](http://www.lamport.org/)
+
 ## leader
 
-The role of a [validator](terminology.md#validator) when it is appending [entries](terminology.md#entry) to the [ledger](terminology.md#ledger).
+Also known as the Proof of History Generator - The [validator](terminology.md#validator) that is selected to append [entries](terminology.md#entry) to the [ledger](terminology.md#ledger) for a given slot. To be accepted as valid, all entries need to be signed with the public key listed within the leader schedule for the slot being verified.
 
 ## leader schedule
 
-A sequence of [validator](terminology.md#validator) [public keys](terminology.md#public-key). The cluster uses the leader schedule to determine which validator is the [leader](terminology.md#leader) at any moment in time.
+A sequence of [validator](terminology.md#validator) [public keys](terminology.md#public-key) matched to all slots in an epoch. Validators shall ensure that a shred is signed by the correct [leader](terminology.md#leader) before confirming its validity to be added to the next block. The leader schedule is calculated from the bank state at the start of the previous epoch.
```

Comment:
"matched" -> "corresponding"?

And no need for that second sentence here.  One doesn't need to know anything about shreds to understand what the leader schedule is.

Reply:
Agreed on changing matched > corresponding. I was drawing a blank when thinking of the word to put there. Corresponding is much better.